e2e/tests/steps_init_test.go

```diff
@@ -59,12 +86,31 @@ func (s *rootSuite) Before() {
 	s.validator = new(validatorPod)
 	s.clientset = clientSet
 	s.validatorKeys = vkmap
+
+	// ADDPR: use pocketk8s to populate
+	s.servicerKeys = map[string]string{
+		// 000 servicer NOT in session
```

Comment:
How do we control which servicer is in a session and which isn't?

Reply:
These tests are based on the genesis, which excludes this servicer and includes the next one: the comment intends to explain what is decided by the genesis file. Updated the comment to try to explain this better.